tsconfig.json

```diff
@@ -1,8 +1,13 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
-    "lib": ["es2017"],
-    "strict": true
-  }
+    /* Additional Checks */
+    "noUnusedLocals": true,
```

Comment:
Mostly for adoption. strict true is great but it would be so painful for our migration. Basically it turns on `noImplicityAny` which is great, except it pretty much makes every import throw. Which sucks. So any file that is imported that isn't converted would break and require us to nearly convert everything at once.

I believe a goal should be to turn strict on. I've turned on every setting that strict does _except_ that one. It sucks because that rule has so much power.